samples/openai_chatgpt/program.py

```diff
@@ -1,40 +1,36 @@
 """This program demonstrates AutoKitteh's OpenAI ChatGPT integration.
 
-This program implements a single entry-point function, which is
-configured in the "autokitteh.yaml" manifest file as the receiver
-of Slack "slash_command" events.
+The program implements a single entry-point function, which is
+configured in the "autokitteh.yaml" manifest file to receive HTTP GET requests.
 
-It sends a couple of requests to the ChatGPT API, and sends the responses
-back to the user over Slack, as well as ChatGPT token usage stats.
+It sends a couple of requests to the ChatGPT API, and returns the responses
+to the user, along with ChatGPT token usage stats.
 
 API details:
 - OpenAI developer platform: https://platform.openai.com/
-- Go client API: https://pkg.go.dev/github.com/sashabaranov/go-openai
+- Python client library: https://github.com/openai/openai-python
 
 This program isn't meant to cover all available functions and events.
 It merely showcases various illustrative, annotated, reusable examples.
-
 """
 
-from autokitteh import openai, slack
+from autokitteh.openai import openai_client
 
 MODEL = "gpt-4o-mini"
 
-chatgpt_client = openai.openai_client("chatgpt_conn")
-slack_client = slack.slack_client("slack_conn")
+chatgpt_client = openai_client("chatgpt_conn")
 
 
-def on_slack_slash_command(event):
-    """https://api.slack.com/interactivity/slash-commands
+def on_http_get(event):
+    """Entry-point function for workflow.
 
-    To use the slash command, simply type `/command-name` in the Slack message input,
-    where `command-name` is the name you have assigned to the command in your app.
-    This command does not require any additional text or arguments.
+    example URL: "http://localhost:9980/webhooks/<webhook_slug>?text=autokittens"
 
     Args:
-        event: Slack event data.
+        event: HTTP event data (including URL query parameters).
     """
-
+    params = event.data.url.query
+    text = params.get("text", "")
```

Comment:
Usually it's much better to initialize variables as close as possible to where they're being used, unless they have some larger meaning or impact on preceding code. Also, as commented in a different PR, no need for `params`.